### DISCUSSION TITLE
request

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [xiaocihua/stack-to-nearby-chests#31] [General]
could you add an option to make it only stack if the item is already in that container

**Comment by xiaocihua** [comment]
Hello @kittenvr !

This mod is supposed to implement the functionality you mentioned. It seems like it's not working properly. Please upload the logs and the list of mods.

**Reply by pizzav-xyz** [reply]
it works now all of a sudden but could you maybe make it also deposit of some catergory like wood 
you could use tags
https://minecraft.wiki/w/Tag

**Reply by xiaocihua** [reply]
Thank you for your suggestion! I will consider adding this feature in future versions.
